external-docs/docs/guides/standard-shape.md

```diff
@@ -0,0 +1,70 @@
+The standard shape extends the [common structure](./shape-data.md#common-structure) by requiring that **each row** has a _measures_ column and a _units_ column; these columns define the measure and unit (of measure) for each row.
+
+The standard shape is most at home where you have a sparse data cube, i.e. there are a large number of possible combinations of dimension values, but very few of them have observed values recorded. If your data cube is dense, then consider using the [pivoted shape](./pivoted-shape.md).
```

Comment:
It'd be good to ensure we have definitions for `sparse` and `dense` in the glossary and provide links to them from here.

Reply:
Done (although wording could probably be improved, possibly with examples)

Reply:
looks good